exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -274,8 +274,7 @@ func (prwe *prwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	if errMarshal != nil {
 		return consumererror.NewPermanent(errMarshal)
 	}
-	buf := make([]byte, len(data), cap(data))
-	compressedData := snappy.Encode(buf, data)
+	compressedData := snappy.Encode(nil, data)
```

Comment:
this is an awesome finding! thanks! Can you add a comment here about it so that nobody thinks about removing this in the future?